tsl/src/continuous_aggs/materialize.c

```diff
@@ -459,7 +468,35 @@ get_materialization_end_point_for_table(int32 raw_hypertable_id, int32 materiali
 		*materializing_new_range = false;
 		return old_completed_threshold;
 	}
-
+	if (max_time < end_time)
+	{
+		int64 maxarg = max_time;
+		/* max_time needs to be adjusted by negative refresh_lag to put it
+		 * into the correct time_bucket. Say we have max_time = 100
+		 * and refresh_lag = -100, time_bucket width = 100
```

Comment:
What happens if someone sets a negative `refresh_lag` which is bigger than the bucket width?